query_base.go

```diff
@@ -65,6 +65,24 @@ var (
 	_ IDB = (*Tx)(nil)
 )
 
+// QueryBuilder is used SelectQuery,UpdateQuery, DeleteQuery for adding new Where conditions
```

Comment:
>for adding new Where conditions

Later, we can add more methods here, for example, `Model() QueryBuilder` and `Table(...string) QueryBuilder`.

For some methods that don't exist in some of the supported queries (SelectQuery/UpdateQuery/DeleteQuery), we could add a no-op/empty implementation that does nothing but implements the interface.